fast64_internal/f3d/f3d_material.py

```diff
@@ -2639,37 +2662,47 @@ class RDPSettings(bpy.types.PropertyGroup):
     )
     aa_en: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Enables anti-aliasing to rasterized primitive edges. Uses coverage to determine edges"
     )
     z_cmp: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Checks pixel Z value against Z-Buffer to test writing"
     )
     z_upd: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Updates the Z-Buffer with the most recently written pixel Z value"
     )
     im_rd: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Enables reading from framebuffer for blending calculations"
     )
     clr_on_cvg: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Only draw on coverage (amount primitive covers target pixel) overflow"
     )
     cvg_dst: bpy.props.EnumProperty(
         name="Coverage Destination",
         items=enumCoverage,
         update=update_node_values_with_preset,
+        description="Changes how coverage (amount primitive covers target pixel) gets retrieved/stored"
     )
     zmode: bpy.props.EnumProperty(
         name="Z Mode",
         items=enumZMode,
         update=update_node_values_with_preset,
+        description="Changes Z calculation for different types of primitives"
     )
     cvg_x_alpha: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Multiply coverage (amount primitive covers target pixel) with alpha and store result as coverage"
     )
     alpha_cvg_sel: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Use coverage (amount primitive covers target pixel) as alpha instead of color combiner alpha"
     )
     force_bl: bpy.props.BoolProperty(
         update=update_node_values_with_preset,
+        description="Always uses blending on. Default blending is conditionally based on depth. Always used when Z Buffering is off"
```

Comment:
Default blending is conditionally based on coverage, not depth--if the pixel is fully covered, it doesn't blend, it just writes the new value. More importantly, however, `force_bl` disables the division step in the blender, meaning B must be equal to 1-A or there may be problems.